README.md

```diff
@@ -1 +1,143 @@
-# math-magician1
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+PROJECT DESCRIPTION -->
+
+# 📖 [math-magician]
+
+> Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] Kindly consider removing this line as this text is directed to the author of this repo and not to the visitor. removing this text will make your repo more professional.